### PR TITLE
Convert  BeneficialOwner and its subclasses to H5

### PR DESCRIPTION
**Pull request by BenGalewsky**
BeneficialOwner is an abstract baseclass. It has two concrete subclasses: PersonBeneficialOwner and OrganizationBeneficialOwner

They both share the same DB table which is discriminating on the column person_ind.

- Added annotations to the three classes
- Added the single table to seed.db
- Removed all three classes from Medicade.hbn.xml
- Added all three classes to persistence.xml

Progress on #36

### CODE REVIEW

File: psm-app/db/seed.sql [hunk 1]
```diff
@@ -1058,3 +1059,29 @@ INSERT INTO screening_schedules(
...
+  person_ind          CHARACTER VARYING(1),
+  ben_type_cd         CHARACTER VARYING(2) REFERENCES beneficial_owner_types(code),
+  oth_type_desc       CHARACTER VARYING(255),
+  subcontractor_name  CHARACTER VARYING(255),
+  own_interest_pct    FLOAT,
+  address_id          BIGINT REFERENCES addresses(address_id),
+  oth_prov_intrst_ind CHARACTER VARYING(255),
+  oth_prov_name       CHARACTER VARYING(255),
+  oth_prov_own_pct    FLOAT,
+  oth_prov_address_id BIGINT,
```
Comment (BenGalewsky) [inline psm-app/db/seed.sql]: It's not used as a foreign key in the code. It is treated as a string. I wonder if its wise to impose that on the db when it's not used that way

File: psm-app/db/seed.sql [hunk 2]
```diff
@@ -1080,3 +1081,32 @@ CREATE TABLE licenses(
...
+  middle_name               TEXT,
+  first_name                TEXT,
+  last_name                 TEXT,
+  ssn                       TEXT,
+  birth_dt                  DATE,
+  hired_at                  DATE,
+  relationship_type_code    CHARACTER VARYING(2)
+    REFERENCES relationship_types (code),
+  ownership_info_id         BIGINT,
+  idx                       INTEGER,
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: What is `idx`? I don't see it referenced anywhere.

File: psm-app/db/seed.sql [hunk 3]
```diff
@@ -1080,3 +1081,32 @@ CREATE TABLE licenses(
...
+  oth_provider_address_id   BIGINT,
+  middle_name               TEXT,
+  first_name                TEXT,
+  last_name                 TEXT,
+  ssn                       TEXT,
+  birth_dt                  DATE,
+  hired_at                  DATE,
+  relationship_type_code    CHARACTER VARYING(2)
+    REFERENCES relationship_types (code),
+  ownership_info_id         BIGINT,
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: What is `ownership_info_id`? I don't see it referenced anywhere.

File: psm-app/db/seed.sql [hunk 4]
```diff
@@ -1080,3 +1081,32 @@ CREATE TABLE licenses(
...
+    REFERENCES beneficial_owner_types (code),
+  oth_type_desc             TEXT,
+  subcontractor_name        TEXT,
+  own_interest_pct          FLOAT,
+  address_id                BIGINT
+    REFERENCES addresses (address_id),
+  oth_provider_interest_ind TEXT,
+  oth_provider_name         TEXT,
+  oth_provider_own_pct      FLOAT,
+  oth_provider_address_id   BIGINT,
```
Comment (BenGalewsky) [inline psm-app/db/seed.sql]: Geez, never mind I was looking at the code in a different branch

File: psm-app/db/seed.sql (hunk 3 above)
Comment (BenGalewsky) [inline psm-app/db/seed.sql]: It's going to be the foreign key to OwnershipInfo entity's oneToMany relationship. That's being tackled in a different branch.
https://github.com/OpenTechStrategies/psm/blob/831e16190889c2c463ce4eb0effc518f1b5ebf09/psm-app/cms-business-process/src/main/resources/META-INF/Medicaid.hbm.xml

Comment (jasonaowen) [inline psm-app/db/seed.sql]: Thanks!

**Pull request merged**